pkg/plugin/sdk/scheduler/scheduler.proto

```
@@ -0,0 +1,84 @@
+syntax = "proto3";
+
+package gameap.plugin.sdk.scheduler;
```

Buf (1.71.0)

[error] Files with package "gameap.plugin.sdk.scheduler" must be within a directory "gameap/plugin/sdk/scheduler" relative to root but were in directory "pkg/plugin/sdk/scheduler".

(PACKAGE_DIRECTORY_MATCH)